docs/src/explanations/connectable.md

```diff
@@ -0,0 +1,444 @@
+
+```scala mdoc:silent
+import chisel3._
+class Parent extends Bundle {
+  val alignedChild = UInt(32.W)
+  val flippedChild = Flipped(UInt(32.W))
+}
+class MyModule0 extends Module {
+  val p = Wire(new Parent)
+}
+```
+
+First, every member is always aligned with themselves:
+ - `p` is aligned w.r.t `p`
+ - `p.alignedChild` is aligned w.r.t `p.alignedChild`
+ - `p.flippedChild` is aligned w.r.t `p.flippedChild`
+
+Next, we list all parent/child relationships. Because the `flippedChild` field is `Flipped`, it changes its aligment relative to its parent. 
+ - `p` is aligned w.r.t `p.alignedChild`
+ - `p` is flipped w.r.t `p.flippedChild`
+
+Finally, we can list all sibling relationships:
+ - `p.alignedChild` is flipped w.r.t `p.flippedChild`
+
+The next example has a nested bundle `GrandParent` who instantiates an aligned `Parent` field and flipped `Parent` field.
+
+```scala mdoc:silent
+import chisel3._
+class GrandParent extends Bundle {
+  val alignedParent = new Parent()
+  val flippedParent = Flipped(new Parent())
+}
+class MyModule1 extends Module {
+  val g = Wire(new GrandParent)
+}
+```
+
+Consider the following alignements between grandparent and grandchildren. An odd number of flips indicate a flipped relationship; even numbers of flips indicate an aligned relationship.
+ - `g` is aligned w.r.t `g.flippedParent.flippedChild`
+ - `g` is aligned w.r.t `g.alignedParent.alignedChild`
+ - `g` is flipped w.r.t `g.flippedParent.alignedChild`
+ - `g` is flipped w.r.t `g.alignedParent.flippedChild`
+
+Consider the following alignment relationships starting from `g.alignedParent` and `g.flippedParent`. *Note that whether `g.alignedParent` is aligned/flipped relative to `g` has no effect on the aligned/flipped relationship between `g.alignedParent` and `g.alignedParent.alignedChild` because alignment is only relative to the two members in question!*:
+ - `g.alignedParent` is aligned w.r.t. `g.alignedParent.alignedChild`
+ - `g.flippedParent` is aligned w.r.t. `g.flippedParent.alignedChild`
+ - `g.alignedParent` is flipped w.r.t. `g.alignedParent.flippedChild`
+ - `g.flippedParent` is flipped w.r.t. `g.flippedParent.flippedChild`
+
+In summary, a member is aligned or flipped w.r.t. another member of the hardware component. This means that the type of the consumer/producer is the only information needed to determine the behavior of any operator. *Whether the consumer/producer is a member of a larger bundle is irrelevant; you ONLY need to know the type of the consumer/producer*.
+
+## Input/Output
+
+`Input(gen)`/`Output(gen)` are coercing operators. They perform two functions: (1) create a new Chisel type that has all flips removed from all recursive children members but structurally equivalent to `gen`, and (2) apply `Flipped` if `Input`, keep aligned (do nothing) if `Output`. E.g. if we imagine a function called `cloneChiselTypeButStripAllFlips`, then `Input(gen)` is equivalent to `Flipped(cloneChiselTypeButStripAllFlips(gen))`.
```

Comment:
you have never explained what "structurally equivalent is". And this sentence seems to imply that the chisel type does include the alignment, but its unclear if alignments need to match for chisel type equivalence.